edx_proctoring/apps.py

```diff
@@ -53,11 +53,14 @@ def ready(self):
         config = settings.PROCTORING_BACKENDS
 
         self.backends = {}  # pylint: disable=W0201
+        not_found = []
         for extension in ExtensionManager(namespace='openedx.proctoring'):
             name = extension.name
             try:
                 options = config[name]
                 self.backends[name] = extension.plugin(**options)
             except KeyError:
-                warnings.warn("No proctoring backend configured for '{}'.  "
-                              "Available: {}".format(name, list(self.backends)))
+                not_found.append(name)
+        if not_found:  # pragma: no branch
```

Comment:
Why not test this?

Reply:
It turns out to be very difficult to test both cases (when some backend settings exist, and when all backend settings exist) because this function gets called once by the django app loading machinery.